### ISSUE TITLE
Making the storage of page-names more intuitive

### ISSUE THREAD

**Issue by luap42** [Jugendhackt/ShitHub#5]
Yesterday we merged #4 to master. But there is still a little problem: The format for storing the field `SITE_NAMES` is dissatisfying. Currently we have a format like the php associative array. Both parsing this format and editing it could be problematic. What should we do eg. with this page title: 
> User => Settings
Maybe this kind of situation is rare but I would prefer doing something different:

In PHP we can do this:
```php
<?php
$foo = "bar";
$bar = "baz";
echo $$foo;
// --> "baz" ($bar)
?>
```
So couldn't we do this?
```env
SITE_LIST = "header, footer, dashboard, upload, dreview, 404, login"
SITE_NAME_header = ""
SITE_NAME_footer = ""
SITE_NAME_dashboard = "Dashboard"
# ...
```
Then we would access a site-name so:
```php
<?php
// ...
// $site = "..."
$site_name = $_ENV["SITE_NAME_$site"]; // Important: Use " and not '!
// ...
?>
```

**Comment by anghenfil** [comment]
I agree, I like your approach. Feel free to implement it

**Comment by luap42** [comment]
Hello, I just implemented it in PR #6.
